### PR TITLE
Telmos/vnet topology

### PR DESCRIPTION
**Pull request by telmosampaio**
@Petertaylor9999 , please check and merge.

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md](https://github.com/mspnp/architecture-center/blob/telmos/vnet-topology/docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/vnet-topology-hub-spoke?branch=pr-en-us-122) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705151953577374-122%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+# Implement a hub-spoke network topology in Azure
+
+This reference architecture shows how to implement a hub-spoke topology in Azure, using the hub as a gateway between Azure and your on-premises datacenter. The use of hub for connectivity into Azure provides the following advantages:
+
+* **Cost savings** by centralizing services that can be shared by multiple workloads, such as network virtual applinaces (NVAs) and DNS servers, in a single location.
+* **Overcome subscriptions limits** by peering virtual networks (VNets) from different ubscriptions to the central hub.
+* **Separation of concerns** between central IT (SecOps, InfraOps) and workloads (DevOps).
+
+Traffic flows between the on-premises datacenter and Azure through an ExpressRoute or VPN gateway connection. This connection is made to a hub virtual network (VNet) in Azure, which in turn is peered to other VNets in the same Azure region, as seen in the piture below.[**Deploy this solution**](#deploy-the-solution).
+
```
Comment (Petertaylor9999) [inline docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md]: Need a space before "deploy this solution".

File: docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+pnp.series.title: Implement a hub-spoke network topology in Azure
+pnp.series.prev: expressroute
+cardTitle: Improving availability
+---
+# Implement a hub-spoke network topology in Azure
+
+This reference architecture shows how to implement a hub-spoke topology in Azure, using the hub as a gateway between Azure and your on-premises datacenter. The use of hub for connectivity into Azure provides the following advantages:
+
+* **Cost savings** by centralizing services that can be shared by multiple workloads, such as network virtual applinaces (NVAs) and DNS servers, in a single location.
+* **Overcome subscriptions limits** by peering virtual networks (VNets) from different ubscriptions to the central hub.
```
Comment (Petertaylor9999) [inline docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md]: add an "s" to "ubscriptions"

File: docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md [hunk 3]
```diff
@@ -0,0 +1,437 @@
+---
+title: Implementing a hub-spoke topology in Azure
+description: >-
+  How to implement a hub-spoke network topoly in Azure.
+author: telmosampaio
+ms.service: guidance
```
Comment (bennage) [inline docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md]: You can omit `ms.service` and `ms.topic`. These are provided via the global metadata.
https://github.com/mspnp/architecture-center/blob/master/docs/docfx.json#L47-L56

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md](https://github.com/mspnp/architecture-center/blob/telmos/vnet-topology/docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/vnet-topology-hub-spoke?branch=pr-en-us-122) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705152151363285-122%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md](https://github.com/mspnp/architecture-center/blob/telmos/vnet-topology/docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/reference-architectures/hybrid-networking/vnet-topology-hub-spoke?branch=pr-en-us-122) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C15%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201705152153157765-122%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**